input/hd63433d.toml

```diff
@@ -240,6 +240,7 @@ author  = "Harrison Nicholls, Tim Lichtenberg"
     [delivery.elements]
         CH_ratio    = 1.0       # C/H ratio in mantle/atmosphere system
```

Comment:
The C content is then the only quantity that is set via different means than ppm. Change this as well? (Perhaps in a new PR).

In addition, I just realise it is not clear here whether this C/H ratio is by mass or by mole fraction. A comment would suffice.

Reply:
Yes, this means that C/H is the only quantity that cannot be specified in ppmw. We could probably make a new issue for this.

I'll fix the description.